pkg/resource/deploy/deploytest/resourcemonitor.go

```diff
@@ -77,11 +83,14 @@ func (rm *ResourceMonitor) RegisterResource(t tokens.Type, name string, custom b
 		IgnoreChanges:        ignoreChanges,
 		Version:              version,
 		Aliases:              aliasStrings,
-	})
+		CustomTimeouts:       &timeouts,
```

Comment:
Should we be passing `nil` here if no custom timeouts were provided? Or are we treating `0` values as "unset"?

Reply:
We are treating them as unset with 0 or ""